src/Storages/ObjectStorage/DataLakes/Iceberg/Utils.cpp

```diff
@@ -741,7 +739,11 @@ std::pair<String, String> parseTransformAndColumn(ASTPtr object, size_t i)
         return parse_function(object);
     }
 
-    auto function_desc = object->children[0]->children[i]->children[0];
+    auto function_desc = object->children[0]->children[i];
```

Comment:
`object->children[0]` here should be `ASTExpressionList`, right? As I remember this is usually checked with `as` before getting its children for ASTFunction